packages/manager/modules/billing/src/orders/purchaseOrders/update-purchase-status/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,9 @@
+{
+  "purchaseOrders_modal_title": "Désactiver",
+  "purchaseOrders_modal_content": "Êtes vous certain de vouloir désactiver votre référence interne ? Si vous désactivez cette référence, elle n'apparaîtra plus sur vos prochaines factures.",
+  "purchaseOrders_modal_primary_action": "Valider",
+  "purchaseOrders_modal_secondary_action": "Annuler",
+  "purchaseOrders_confirmation_desactivation_REFERENCE_ORDER_success": "Votre référence interne a bien été désactivé.",
+  "purchaseOrders_confirmation_desactivation_PURCHASE_ORDER_success": "Votre purchase order a bien été désactivé.",
```

Comment:
désactivée

Reply:
Is it intended to set mix of french and english here?

Reply:
This is currently being considered by the PU
Not blocking for Q&C as it is content

Reply:
I have been confirmed by PU. We keep english term. Purchase Order is a recognised term in the accounting domain (the term is masculine).